llvm/include/llvm/Analysis/DominanceFrontier.h

```diff
@@ -39,7 +39,7 @@ class raw_ostream;
 template <class BlockT, bool IsPostDom>
 class DominanceFrontierBase {
 public:
-  using DomSetType = std::set<BlockT *>;                // Dom set for a bb
+  using DomSetType = SetVector<BlockT *>;               // Dom set for a bb
```

Comment:
Can you add a comment explaining the choice of  `SetVector`?

Reply:
done